packages/console/src/containers/ConsoleContent/index.tsx

```diff
@@ -137,6 +140,12 @@ function ConsoleContent() {
               <Route path="link-email" element={<LinkEmailModal />} />
               <Route path="verification-code" element={<VerificationCodeModal />} />
             </Route>
+            {isCloud && (
+              <Route path="settings">
+                <Route index element={<Navigate replace to={TenantSettingsTabs.Settings} />} />
+                <Route path=":tab" element={<TenantSettings />} />
```

Comment:
We can use the `OutletContext` to specify a specific path.
Like:
```ts
<Route path="roles">
  <Route index element={<Roles />} />
  <Route path="create" element={<Roles />} />
  <Route path=":id" element={<RoleDetails />}>
    <Route index element={<Navigate replace to={RoleDetailsTabs.Settings} />} />
    <Route path={RoleDetailsTabs.Settings} element={<RoleSettings />} />
    <Route path={RoleDetailsTabs.Permissions} element={<RolePermissions />} />
    <Route path={RoleDetailsTabs.Users} element={<RoleUsers />} />
  </Route>
</Route>
```

Reply:
The sign-in-experience page use `:tab` is because we need to save data across all tabs, so we cannot use the `OutletContext` there.